setup.cfg

```diff
@@ -0,0 +1,60 @@
+[metadata]
+name = tszip
+author = Tskit Developers
+author_email = admin@tskit.dev
+license = MIT
+description = Compression utilities for tree sequences
+long_description_content_type = text/markdown
+long_description = file: README.md
+url = http://pypi.python.org/pypi/tszip
+project_urls =
+    Documentation = https://tszip.readthedocs.io/en/latest/
+    Changelog = https://github.com/tskit-dev/tszip/blob/main/CHANGELOG.rst
```

Comment:
Better to point to https://tszip.readthedocs.io/en/latest/changelog.html because we might rename the changelog to md, which would break the link. In principle RTD will redirect if/when we ever change the link.

Reply:
Good point, fixed